modules/ROOT/pages/authentication-authorization/load-privileges.adoc

```diff
@@ -79,4 +87,35 @@ SHOW ROLE roleLoadAllData PRIVILEGES AS COMMANDS
 a|Rows: 1
 |===
 
-The `LOAD ON ALL DATA` privilege is granted to the `PUBLIC` role by default.
\ No newline at end of file
+The `LOAD ON ALL DATA` privilege is granted to the `PUBLIC` role by default.
+
+[[access-control-load-cidr]]
+== The `CIDR` privilege
+
+_This feature was released in Neo4j 5.16._
+
+The load privilege on `CIDR cidr` enables or disables loading data from the given IPv4 or IPv6 CIDR range.
+If granted, the user can load data from sources in the given CIDR range.
+If missing or denied, no data can be loaded from sources in the given CIDR range.
+
+For example, the below query denies users with the role `roleLoadCidr` the ability to load data with `LOAD CSV` from `127.0.0.1/32`:
+
+[source, cypher, role=noplay]
+----
+DENY LOAD ON CIDR "127.0.0.1/32" TO roleLoadCidr
+----
+
+The below query lists all privileges for the role `roleLoadCidr` as commands:
+
+[source, cypher, role=noplay]
```

Comment:
From  slack: `role=noplay` is no longer needed nor has any effect on code blocks

Reply:
I think `role=noheader` also does nothing.